notebooks/time-to-merge-prediction/workshop/intro_to_jupyternotebooks.ipynb

```diff
@@ -0,0 +1,1050 @@
+{
```

Comment:
<p>can you re-run the notebook so that the error in this cell and the next goes away?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/573/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
fixed